### PR TITLE
John conroy/webpack dev server

### PR DESCRIPTION
**Pull request by john-conroy**
Typeface-inter is causing a lot of the bundle bloat (~50% of the bundle size).

### CODE REVIEW

**Comment by john-conroy** [comment]
@mccalluc , can you take a look at quickstart.sh?

**Comment by mccalluc** [comment]
@john-conroy :
These lines in test.sh were clearing the TODOs:
```
if [ ! -z "$TRAVIS" ]; then
  ./quick-start.sh || sed -i 's/TODO/FAKE/' context/instance/app.conf
fi
```
Not clear why they're not working right now... also: quickstart.sh was always kind of shoddy, so don't feel that it can't be significantly revamped, if that's the easier route.

I'm going to do the Tuesday morning redeploy now, but can look at this some more later, if it's useful.

**Comment by john-conroy** [comment]
I reverted `quick-start.sh` to its old form and created a new script, `dev-start.sh`, to handle starting the dev server.

File: CHANGELOG-webpack-dev-server.md [hunk 1]
```diff
@@ -0,0 +1,3 @@
+- Add webpack dev server.
+- Split webpack config into separate files for each env.
+- Add webpack bundle analyzer/visualizer.
```
Comment (mccalluc) [inline CHANGELOG-webpack-dev-server.md:3]: Can you add a note in the readme about how/when this would be used?

File: README.md [hunk 2]
```diff
@@ -44,12 +44,9 @@ After checking out the project, cd-ing into it, and setting up a Python3.7 virtu
 - `wget https://raw.githubusercontent.com/hubmapconsortium/prov-vis/master/src/schema.json -O node_modules/@hubmap/prov-vis/es/schema.json`
 - (Note: Error with @hubmap-prov-vis dependency. Manually add
 the schema.json until [this issue is resolved](https://github.com/hubmapconsortium/portal-ui/issues/139).)
-- `npm run dev-build`
 - `./quick-start.sh`,
 - update `app.conf` with the Globus client ID and client secret, and the Elasticsearch endpoint,
-- `./quick-start.sh` again,
-and visit [localhost:5000](http://localhost:5000), or append `?react`
-to a `/dataset` url to see the React version.
+- `./dev-start.sh` to run the webpack dev and flask servers and visit [localhost:5001](http://localhost:5001).
```
Comment (mccalluc) [inline README.md:49]: I think 5001 is the new react dev server... Won't folks still go to flask, and then the requests for JS from there are handled by the react dev server?

Comment (mccalluc) [reply README.md:49]: Oh: got down to this:
```
    proxy: {
      '!(/static/public//**/**.*)': {
        target: 'http://localhost:5000/',
      },
    },
```
The react will proxy to the flask? Adding a sentence here to explain that would be good.

File: CHANGELOG-webpack-dev-server.md (hunk 1 above)
Comment (john-conroy) [reply CHANGELOG-webpack-dev-server.md:3]: Assuming you meant the webpack bundle visualizations, done.

**Pull request merged**